### ISSUE TITLE
Missing podref field in ippools.whereabouts object

### ISSUE THREAD

**Issue by tardich** [k8snetworkplumbingwg/whereabouts#112]
Hi, in my cluster at the office, when assigning an IP through whereabouts, I'm seeing this IP reference in ippools.whereabouts object as:

```
spec:
  allocations:
    "199":
      id: 988ec2858be5bba9ca8085313bd54df729ad976721baa5d70b0bcfaa5a6a546f
      podref: powerdns-master/powerdns-master-powerdns-authoritative-645dd4cfc6-hqkgt
```

In my cluster at home, an IP assigned through whereabouts only shows:

```
spec:
  allocations:
    "254":
      id: 2329905a30130454f502c64ad531af3bc3fb38395708f521161b0ea8373001c5
```

The podref field does not appear, making it hard to find out where this IP is assigned without looking at all network-attachment-definitions. I don't recall having installed whereabouts any differently on these two clusters. 

Any guidance into fixing this would be welcome.

**Comment by maiqueb** [comment]
I think this issue was fixed by https://github.com/k8snetworkplumbingwg/whereabouts/pull/116

Please close the issue if you agree.

**Comment by maiqueb** [comment]
@dougbtv this can also be closed IMO.

**Comment by maiqueb** [comment]
I really think this issue was already fixed in #116. Closing for lack of replies confirming it.
